package.json

```diff
@@ -3,8 +3,7 @@
   "displayName": "Betaflight - Blackbox Explorer",
   "description": "Crossplatform blackbox analitics tool for Betaflight flight control system.",
   "version": "3.5.0",
-  "main": "main_nwjs.html",
-  "bg-script": "background.js",
+  "main": "index.html",
```

Comment:
It seems that not. If I remember correctly, we added this to the Configurator because it had problems opening/saving files to disk. The Blackbox must use something different because it seems to be working. At some moment we need to revisit the Configurator part.